index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  Hello Microverse
```

Comment:
Please place this text inside an H1 tag which should also be inside a header, I will give you an example of how should the HTML structure be according to the project requirements:

```html
<header>
  <h1>
    Your text here
  </h1>
</header>
```